db/fixtures/miq_product_features.yml

```diff
@@ -5129,6 +5137,10 @@
         :description: Suspend VM
         :feature_type: control
         :identifier: vm_suspend
+      - :name: Pause
+        :description: Pause VM
+        :feature_type: control
+        :identifier: vm_pause
```

Comment:
Interesting that this wasn't there.

@h-kataria can we borrow you to review the miq_product_features.yml changes ?  Thanks.